create-neon/data/templates/.gitignore.hbs

```diff
@@ -0,0 +1,4 @@
+target
+index.node
+**/node_modules
+**/.DS_Store
```

Comment:
Should we include anything else from the node `.gitignore` template? https://github.com/github/gitignore/blob/master/Node.gitignore

A lot of them are for specific packages, but `npm-debug.log*` might be nice.